src/tests/ftest/soak/stress_48h.yaml

```diff
@@ -264,7 +263,48 @@ macsio_stress:
     mount_dir: "/tmp/daos_dfuse/macsio/"
     disable_caching: true
     thread_count: 8
-    cores: '0-3'
+    cores: '0-7'
+datamover_stress:
+  job_timeout: 10
+  nodesperjob:
+    - 1
+    - 4
+    - 8
+  taskspernode:
+    - 16
+    - 32
+  oclass:
+    - ["SX","SX"]
+    - ["EC_2P1GX", "RP_2GX"]
+    #  - ["EC_4P1G1", "RP_2GX"]
+    - ["EC_4P2GX", "RP_3GX"]
+    #  - ["EC_8P2G1", "RP_3GX"]
+    #  - ["EC_16P2GX", "RP_3GX"]
+  dcp:
+    bufsize: 4M
+    chunksize: 128M
```

Comment:
In case `dcp --help` isn't clear:
- bufsize - The IO read/write size. Similar to IOR transfer_size
- chunksize - The size of each chunk of work. In this case, 128M means the file is split into chunks of 128M and each process gets an equal number of chunks.

Caveat to `chunksize` by example:
```
mpirun -np 32*8 ior -w -k -t 1M -b 1M ...
# Creates file of size 256M - no problem

mpirun -np 32*8 dcp  --chunksize 128M ...
# File is split into chunks of 128M
# rank0 works on 128M
# rank1 works on 128M
# remaining ranks work on nothing
```